.github/workflows/drivers.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Check if driver tests should run
         id: check-branch-and-files-changed
         run: |
-          IS_MASTER_OR_RELEASE="${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release-') }}"
+          IS_MASTER_OR_RELEASE="${{ github.ref_name == 'master' || startsWith(github.ref_name, 'release-') }}"
```

Comment:
I think this behavior will be unchanged. It was working and it will keep working.